framer/EventEmitter.coffee

```diff
@@ -1,3 +1,94 @@
-EventEmitter = require "eventemitter3"
```

Comment:
@koenbok It worries me a little that we're dropping a proven eventing lib and rolling with our own instead. Are you sure that we're not signing up for unforeseen bugs and performance issues?